src/main/java/com/osunkwo/taskmaster/TaskMasterController.java

```diff
@@ -3,23 +3,33 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
 import java.util.List;
 import java.util.UUID;
 
 @RestController
+@CrossOrigin
 public class TaskMasterController {
 
```

Comment:
You can put `@Autowired` here and then you don't need the
`@Autowired
    TaskMasterController(S3Client s3Client){
        this.s3Client = s3Client;
    }`